src/main/java/frc/team2412/robot/subsystem/MonitoringSubsystem.java

```diff
@@ -0,0 +1,72 @@
+package frc.team2412.robot.subsystem;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class MonitoringSubsystem extends SubsystemBase {
+    private final ShuffleboardTab tab = Shuffleboard.getTab("Power Monitoring");
+    private final NetworkTableEntry totalPower, totalCurrent, currentTemperature,
+            channel0current, channel1current, channel2current, channel3current, channel4current, channel5current,
+            channel6current, channel7current,
+            channel8current, channel9current, channel10current, channel11current, channel12current, channel13current,
+            channel14current, channel15current;
```

Comment:
Not sure this is necessary anymore, as the driver station logs include the current values for all the PDM channels already.